### PR TITLE
Twitter importer

### PR DESCRIPTION
**Pull request by alejandrodob**
Adds Twitter importer. Review also submodules: https://github.com/PyBossa/pybossa-default-theme and https://github.com/PyBossa/crowdcrafting-theme/pull/292

### CODE REVIEW

File: doc/customizing.rst [hunk 1]
```diff
@@ -753,6 +753,19 @@ Instead, get the App key you will be given and add it to your settings_local.py:
...
+
+If you already have enabled the Twitter authentication, then the Twitter task
+importer will be enabled too. Otherwise, you will need to create an application
+in Twitter_ and copy and paste the **Consumer key and secret** into the next
+variables: `TWITTER_CONSUMER_KEY`_ and `TWITTER_CONSUMER_SECRET`_ and uncomment
+them.
+
+.. _Twitter: https://dev.twitter.com/
+.. note::
+    This will also enable Twitter authentication.
```
Comment (therealmarv) [inline doc/customizing.rst]: I would write: `This will also enable PyBossa's Twitter login`

Comment (alejandrodob) [inline doc/customizing.rst]: yep, that sound much better, fixing it!

**Comment by teleyinex** [comment]
:cake:

**Comment by therealmarv** [comment]
:cake: finally :)

**Pull request merged**